README.md

```diff
@@ -1,2 +1,56 @@
 # sustainable-society
 Collection of documents to use when needing an advice regarding sustainability 
+
+## Food
+-  Buy local and seasonal food, no need to buy summer food in winter, or if you do have cravings for it buy (or do it yourself) pickled or in naother way preserved version
+
+- Avoid supermarkets for most of the things, buy there only the things that are not available locally
+
+- Eat less meat, it's more sustainable and healthier
```

Comment:
Depends on where and how you get your meat. But in general, it's unsustainable to eat meat and meat products every day. Also, lots of meat products are mass produced so it's best to avoid them too.